sdk/compute/Azure.ResourceManager.Compute/src/Customize/Extensions/ComputeExtensions.cs

```diff
@@ -6,6 +6,7 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Azure.Core;
+using Azure.ResourceManager.Compute.Mocking;
```

Comment:
I think we are not explicitly using anything from this namespace, maybe we could do a using clean up to remove this.